sqldelight-gradle-plugin/src/test/fixtures/duplicate-table-name/failure.txt

```diff
@@ -0,0 +1,5 @@
+Table2.sq line 1:13 - Table already defined with name test
+  1		CREATE TABLE test (
+   		             ^^^^
+  2		  _id LONG NOT NULL PRIMARY KEY
+  3		)
```

Comment:
Yeah. I had to use sublime text. I would just do spaces and the alignment yourself :+1: